frontend/src/components/HacknightsParticipants.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <v-container fluid text-xs-center>
+    <v-row align="center" justify="center">
+      <v-col class="d-flex" cols="12" sm="6">
+        <v-combobox
+          v-model="selectedParticipants"
+          outlined
+          :items="
+            getParticipants.filter(
+              participant =>
+                !getHacknight.participants.filter(y => y.id === participant.id)
+                  .length
+            )
+          "
+          item-text="email"
+          item-value="email"
+          :search-input.sync="search"
+          hide-selected
+          label="Add participants"
+          multiple
+          persistent-hint
+          small-chips
+          clearable
+          deletable-chips
+          append-outer-icon="mdi-cloud-upload"
+        >
+          <template v-if="noData" v-slot:no-data>
+            <v-list-item>
+              <v-list-item-content>
+                <v-list-item-title>
+                  No results matching "<strong>{{ search }}</strong
+                  >".
```

Comment:
That's very strange line break

Reply:
When i've changed it, prettier reformatted it as above again :/